src/WinXP/index.js

```diff
@@ -32,6 +36,8 @@ const initState = {
   focusing: FOCUSING.WINDOW,
   icons: defaultIconState,
   selecting: false,
+  contextMenu: false,
+  contextPos: null,
```

Comment:
It's ok to either separate these to two properties like you did or combine into one; But please don't be cheap on characters, `contextPosition` would be more readable 😉 

Other than that, this file looks great.

Reply:
Didn't think about it 🤦‍♂️
combined into one.